### PR TITLE
`<DropdownLayout/>` - stylable states

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
removing `classnames` and using stylable states instead 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5725/

File: src/DropdownLayout/DropdownLayout.driver.js [hunk 1]
```diff
@@ -124,7 +144,9 @@ const dropdownLayoutDriverFactory = ({ element }) => {
       values(optionElements.childNodes).map(option => option.textContent),
 
     markedOption: async () => {
-      const hoveredOption = optionElements.querySelector(`.${styles.hovered}`);
+      const hoveredOption = optionElements.querySelector(
+        `[${DATA_OPTION.HOVERED}="true"]`,
```
Comment (EgoziE) [inline src/DropdownLayout/DropdownLayout.driver.js:148]: I think it will work without `="true"` 🤔

File: src/DropdownLayout/DropdownLayout.st.css [hunk 2]
```diff
@@ -3,6 +3,11 @@
   -st-named: B10, B20, B50, D10, D20, D50, D60, D70, D80;
 }
 
+:import {
+  -st-from: "../Foundation/stylable/mixins/calc.js";
+  -st-default: calc;
+}
+
```
Comment (EgoziE) [inline src/DropdownLayout/DropdownLayout.st.css:10]: Isn't calc a basic css function?
I guess it doesn't work with stylable

Comment (TalZa1810) [reply src/DropdownLayout/DropdownLayout.st.css:10]: https://github.com/wix/wix-style-react/pull/5721#discussion_r457960307

**Pull request merged**